client/src/pages/Login.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom';
+
+const Login = () => {
+
+  const [email, setEmail] = useState('');
+
+  const handleLogin = (event) => {
+    event.preventDefault();
+    console.log('button clicked', event.target);
+  }
+
+  const handleEmailChange = (event) => {
+    console.log(event.target.value);
```

Comment:
logs in the handleLogin make sense for now but these in the onChange should be removed

Reply:
> It works right now and it can be merged.
> I'd like to challenge you to see if you can actually refactor the Form into its own component and share that component across both pages. This can help with reducing duplicate code as this page gets more fleshed out

Hi Tony,

So you'd recommend to merge this pull request and create a new pull request with the form refactoring?